### PR TITLE
Adds/removes Cards Against Spess cards.

### PR DESCRIPTION
**Pull request by harmonyn**
## About The Pull Request

people actually playing the game inspired me to do this, please offer suggestions

## Why It's Good For The Game

the deck felt a bit thin

## Changelog
:cl: bandit
tweak: More Cards Against Spess cards are available!
/:cl:

### CODE REVIEW

File: strings/cas_white.txt [hunk 1]
```diff
@@ -1,10 +1,11 @@
 Those motherfucking carp.
 Having sex in the maintenance tunnels.
 Space 'Nam.
+Space capitalism.
 Space lesbians.
 Space NEETs.
 Space Hitler.
-Space Asshole.
+Space assholes.
```
Comment (Dorsisdwarf) [inline strings/cas_white.txt]: boooooo

Comment (Dorsisdwarf) [reply strings/cas_white.txt]: To clarify, I dislike this change and think that Space Asshole is better than genericising it

Comment (bgobandit) [reply strings/cas_white.txt]: Space Asshole fits into fewer black prompts, is the thing

**Comment by cacogen** [comment]
some of the things you've removed i don't understand but then i never play this or see anyone play it so who cares

**Comment by bgobandit** [comment]
generally if I removed something it is because of the incredibly objective metric of my deeming it unfunny

**Pull request merged**